todo_app/app.py

```diff
@@ -1,59 +1,50 @@
-from calendar import c
```

Comment:
There are a few changes in this PR seeming to swap entire files - line ending change snuck through?

Reply:
Oh dear, so there are. Sorry, that must have made it a nightmare to review!